README.md

```diff
@@ -97,18 +98,27 @@ configuration. The following values are supported:
   `refs/heads/$branch/$head`, where `$branch` is the mercurial branch name
   and `$head` is the full changeset sha1 of that head.
 
-- `all` (default): in this mode:
-  - bookmarks are exposed as `refs/heads/bookmarks/$bookmark`,
-  - branch heads are exposed as `refs/heads/branches/$branch/$head` (where
-    `$head` is the full changeset sha1 of the head),
-  - except the branch tip, which is exposed as
-    `refs/heads/branches/$branch/tip`.
-
-These refs styles may also be combined in a comma-separated list. For example,
-a value of `bookmarks,heads,tips` is the same as `all`. A value of
-`bookmarks,tips` exposes the bookmarks and branch tips as
-`refs/heads/bookmarks/$bookmark` and `refs/heads/branches/$branch`,
-respectively.
+When these values are used in combinations, the branch mappings are varied
+accordingly to make the type of each remote ref explicit and to avoid name
+collisions.
+
+- `bookmarks,heads`: in this mode, bookmarks are exposed as
```

Comment:
Nitpick: the wording kind of implies this is the only way to arrange the combination. Maybe something like "- When combining `bookmarks` and `heads`, bookmarks are exposed..."

Reply:
Sure, that looks clearer, done